test-scripts/properties-tests.sh

```diff
@@ -724,6 +762,7 @@ function properties_tests {
     properties_get_with_namespace_raw_format
     properties_secure_namespace_set
     properties_secure_namespace_delete
+    # properties_resources_apply
```

Comment:
I guess you intend to un-comment this once the server-side changes are available ?

Reply:
yes, will be changed in a second pr